docker-compose.yml

```diff
@@ -35,23 +35,26 @@ services:
     networks:
       - public
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:7.10.2
+    image: opensearchproject/opensearch:latest 
```

Comment:
Seems like we may want to pin this to whichever version of open search we plan to have in staging/production

Reply:
Wanted to try and match the rolling minor updates that do occur for Opensearch in AWS (well usually configured). But specifying major version is helpful, so I've changed to specify 2 tag which includes minor version changes